osmaxx-py/osmaxx/excerptexport/tests/test_models.py

```diff
@@ -40,3 +46,37 @@ def test_geometry_has_clockwise_exterior_ring_starting_at_south_west(self):
         BBoxBoundingGeometry.create_from_bounding_box_coordinates(1.1, 2.2, 3.3, 4.4)
         self.assertEqual(BoundingGeometry.objects.first().bboxboundinggeometry.geometry.exterior_ring[:],
                          [(4.4, 3.3), (4.4, 1.1), (2.2, 1.1), (2.2, 3.3), (4.4, 3.3)])
+
+
+class ExtractionOrderTestCase(TestCase):
+    user = None
+    excerpt = None
+    extraction_configuration = None
+
+    def setUp(self):
+        self.user = User.objects.create_user('user', 'user@example.com', 'pw')
+        self.excerpt = models.Excerpt.objects.create(
+            name='Neverland',
+            is_active=True,
+            is_public=False,
+            owner=self.user,
+            bounding_geometry=models.OsmosisPolygonFilterBoundingGeometry.objects.create(
+                polygon_file=SimpleUploadedFile('in_memory_file.poly', b'the file content (not a real .poly file)')
+            )
+        )
+        self.extraction_configuration = {
+            'formats': ['txt'],
+            'options': {
+                'detail_level': 'standard'
+            }
+        }
+
+    def test_create_and_restore_extraction_configuration(self):
+        extraction_order_id = models.ExtractionOrder.objects.create(
+            excerpt=self.excerpt,
+            orderer=self.user,
+            extraction_configuration={'formats': ['txt'], 'options': {'detail_level': 'standard'}}
+        ).id
+        extraction_order = models.ExtractionOrder.objects.get(pk=extraction_order_id)
+        self.assertEqual(extraction_order._extraction_configuration, json.dumps(self.extraction_configuration))
```

Comment:
Aren't you testing implementation details instead of contracts with that first assertion? I guess `ExtractionOrder._extraction_configuration` isn't meant to be used from outside the class, so the second assertion should suffice to ensure the class' contract.